lib/data/file/backend.js

```diff
@@ -1,93 +1,110 @@
 import { errors } from 'arsenal';
 import crypto from 'crypto';
-import stream from 'stream';
 import fs from 'fs';
+import config from '../../Config';
+import Logger from 'werelogs';
 
```

Comment:
Please use in order:
core dependencies
nl
external dependencies
nl
internal dependencies